server/utils/logger.js

```diff
@@ -0,0 +1,15 @@
+const info = (...params) => {
```

Comment:
Although I can understand your code, consider adding some comments for readability. It is good practice because some time later when you revisit this code, the comment you left will make it easier for you to quickly remember why you wrote the code and what it does.
Also, we have teammates and I personally want to see some comments so that I can quickly understand what you're doing right off the bat. Consider, other teammates too. Readability is one of the pillars of good code and leaving comments for yourself and team members is one way to implement readability.
So, please write comments for all the changes you made except the obvious ones like importing a module.